### ISSUE TITLE
System Usability Survey - Internal

### ISSUE THREAD

**Issue by rajpersram** [bcgov/cas-cif#27]
Task: Plan System Usability Survey with internal CIF users to understand overall success and pain points in their workflow.

** Acceptance Criteria: **
- [x] Plan testing
- [x] Create survey
- [x] Send to CIF team members 
- [x] Analyze information (Note: this could be split into different ticket)
- [x] Plan how feedback received will support/align with CIF team.

**Comment by rajpersram** [comment]
@rajpersram to break out the last three item's details.

**Comment by Sriram-pc** [comment]
To be repurposed into System Usability Survey Ticket

**Comment by LindsayMacfarlane** [comment]
@rajpersram - reminder to update this ticket to be for the system usability survey. We can then pull into this sprint.

**Comment by rajpersram** [comment]
6 anonymous surveys were completed by CIF staff, with the results attached. This is our benchmark, so we can't draw too many conclusions from this until a relative future result is gathered (which should be the exact same questionnaire but for the new web-app).

[https://app.zenhub.com/files/402862646/6f5ae2e4-ffcc-4032-bf5a-eac949dcb482/download](https://app.zenhub.com/files/402862646/6f5ae2e4-ffcc-4032-bf5a-eac949dcb482/download)

**Comment by LindsayMacfarlane** [comment]
Thank you @rajpersram. I add a copy of the survey results (from link above) into our [CIF Internal Users Channel](https://teams.microsoft.com/l/file/59CBD38C-8D72-4EF1-A477-386FA670DD26?tenantId=6fdb5200-3d0d-4a8a-b036-d3685e359adc&fileType=pdf&objectUrl=https%3A%2F%2Fbcgov.sharepoint.com%2Fteams%2F00608-CIFInternalUsers%2FShared%20Documents%2FCIF%20Internal%20Users%2FAnalytics%2C%20Metrics%20and%20KPIs%2FSystem%20Usability%20Survey%2FSystem%20Usability%20Survey_Nov%202021.pdf&baseUrl=https%3A%2F%2Fbcgov.sharepoint.com%2Fteams%2F00608-CIFInternalUsers&serviceName=teams&threadId=19:668a87bc0b7e43b0a8948310465bd928@thread.tacv2&groupId=29821293-3244-455d-beae-e01d27762cfe).

**Comment by rajpersram** [comment]
In terms of what stood out, most of the answers were in the middle of the road in terms of the CIF's team's usability of their current system. I had expected answers to score lower in general but I think their answers are normal in the context that they're already used to the system they have, so there is a measure of acceptance for the status quo. There are no wrong answers for this benchmark, but it is just that; A starting point which ultimately needs another survey to follow up on the upcoming web-app. Those score will represent how we've evolved and ideally simplified their processes. Those scores can then be used to establish a rough ROI but we should collect more performance metrics such as what we covered in the KPI meeting with Lindsay, Jaimie, Taylor and Ram. 

As for a plan to support/align, the results of this survey are recorded on the Miro 'CIF Metrics' board with a follow up plan and corresponding user story in ticket 118 for the next survey.
